Code/Framework/AzFramework/AzFramework/Quality/QualityCVarGroup.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Memory/Memory_fwd.h>
+#include <AzCore/Console/IConsole.h>
+#include <AzCore/Console/ConsoleFunctor.h>
+#include <AzCore/std/string/string.h>
+#include <AzCore/std/string/string_view.h>
+#include <AzCore/Preprocessor/Enum.h> 
+
+namespace AzFramework
+{
+    AZ_ENUM_CLASS(QualityLevels,
+        (LevelFromDeviceRules, -1),
+        (DefaultQualityLevel, 0)
+    );
+    AZ_DEFINE_ENUM_ARITHMETIC_OPERATORS(QualityLevels);
+    AZ_DEFINE_ENUM_RELATIONAL_OPERATORS(QualityLevels);
+
+    // QualityCVarGroup wraps an integer CVar for a quality group that, when set,
+    // notifies the QualitySystemBus to load all the settings for the group and the
+    // requested level
+    //
+    // Example:
+    // A q_shadows group CVar could be created with 4 levels (low=0,medium=1,high=2,veryhigh=3)
+    // When the level of q_shadows is set to 2 the QualitySystem will be notified
+    // and apply all console settings defined in the SettingsRegistry for that
+    // group at level 2 (high).
+    //

```

Comment:
This would also be a good place to document that the system will fall back to the highest-valued quality level if a level is requested that's higher than the highest value.

I'm not sure if the system has any assumptions on ordering (low to high vs high to low), but if so, that should get documented too.